client/my-sites/stats/feedback/style.scss

```diff
@@ -4,5 +4,45 @@
 	border-radius: 5px; // stylelint-disable-line scales/radii
 
 	padding: 20px;
-	margin: 32px;
+
+	@media (max-width: 782px) {
+		border-radius: 0;
+		border-top: none;
+		border-left: none;
+		border-right: none;
+	}
+
+	display: flex;
+	flex-direction: column;
+
+	@media (min-width: 1340px) {
+		flex-direction: row;
+		justify-content: space-between;
+		align-items: center;
+	}
+}
+
+.stats-feedback-card__cta {
+	@media (max-width: 1340px) {
+		margin-bottom: 12px;
+	}
+}
+
+.stats-feedback-card__actions {
+	display: flex;
+	flex-direction: column;
+	// margin: 20px 0;
```

Comment:
I personally would prefer not to have code commented out in shipped code. I understand it's not finished yet, but still it takes up space and not much meaning.